### PR TITLE
Fix backstage secret, and backstage permission for configmap

### PR DESCRIPTION
**Pull request by allamand**

### CODE REVIEW

File: terraform/scripts/backstage/output.yaml
Comment (nabuskey) [inline terraform/scripts/backstage/output.yaml:1]: Let's get rid of this file

**Review by nabuskey — COMMENTED** [review]
Thank you very much for this PR. Could you please rebase and remove the empty file? Otherwise, it looks good.

File: terraform/scripts/backstage/output.yaml
Comment (allamand) [reply terraform/scripts/backstage/output.yaml:1]: ok

**Review by nabuskey — APPROVED** [review]
LGTM

**Pull request merged**